playbook/app/pb_kits/playbook/pb_paginate/paginate.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbPaginate
+    class Paginate < ::Playbook::KitBase
+      prop :stragety # Need to add protection
```

Comment:
Let's correct the spelling of strategy here. 
```suggestion
      prop :strategy # Need to add protection
```